modmesh/gui/naca.py

```diff
@@ -112,10 +164,20 @@ def runmain():
     A simple example for drawing a couple of cubic Bezier curves.
     """
     w = core.WorldFp64()
-    draw_naca4(w, number='0012', npoint=101, fac=5.0, off_x=0.0, off_y=2.0,
-               open_trailing_edge=False, cosine_spacing=True)
+    draw_naca4(
+        w,
```

Comment:
This one-argument-a-line style takes too many line.  Please keep the original terse style.